src/components/content/heroes/aurora/aurora-components.ts

```diff
@@ -0,0 +1,54 @@
+import styled, { css } from 'styled-components';
+import { WeatherType } from './weather-easter-egg/weather-types';
+
+const BACKGROUND_LAYER_Z = -2;
+const FOREGROUND_LAYER_Z = -1;
+
+interface AuroraBackgroundProps {
+  source: string;
+  weather: WeatherType;
+}
+
+export const AuroraBackground = styled.div<AuroraBackgroundProps>`
+  background: ${(props) => {
+    switch (props.weather) {
+      case WeatherType.Sunny:
+        return '#3E61EE';
+      case WeatherType.Rainy:
+        return '#76809b';
+      case WeatherType.SlightlyCloudy:
+        return '#5A6FC1';
```

Comment:
That background makes it super hard to read for slightly cloudy. The sun one is much better to read. It's an easter egg and the visual is more important, but it's something that popped my eye.

<img width="576" alt="image" src="https://github.com/satellytes/satellytes.com/assets/1701755/be8366e7-e34c-4280-a591-6d02d9b40341">